plugins/bookmarks.js

```diff
@@ -1,10 +1,13 @@
-// @author         ZasoGD
+// @author         ZasoGD / Aradiv
```

Comment:
Let's reserve this field for original author, and do not change unless you rewrite plugin from scratch.

You may want to introduce separate field `@coauthor` or `@collaborator`, but then we should keep same approach through all plugins, which currently is not in case.

Reply:
While this is a different issue, I added myself to a script internal history (which I still prefer), but it was not excepted either.
As scripts tend to be privately improved, extended or modified in a different ways and get shared outside of GIT and version numbers have their own lives anyway, we need to address this somehow. @johnd0e please create a separate discussion issue on this.